pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -73,6 +74,7 @@ func AddToManager(mgr manager.Manager,
 		pullSecretGetter:         pullSecretGetter,
 		successfulImportsCounter: successfulImportsCounter,
 		failedImportsCounter:     failedImportsCounter,
+		deniedRegistries:         deniedRegistries,
```

Comment:
can we rename this to `forbiddenRegistries` (and everywhere else as well)? `denied` means it was denied in the past, not that it is currently forbidden

Reply:
Done.